.github/workflows/build-cloudberry.yml

```diff
@@ -326,6 +326,12 @@ jobs:
               },
               {"test":"ic-cbdb-parallel",
                "make_configs":["src/test/regress:installcheck-cbdb-parallel"]
+              },
+              {"test":"ic-orca-parallel",
+               "make_configs":["src/test/regress:installcheck-orca-parallel"],
+               "pg_settings":{
+                 "optimizer_enable_parallel_append":"true"
+                }
```

Comment:
That would be better to add the same test to the [build-deb-cloudberry.yml](https://github.com/apache/cloudberry/blob/main/.github/workflows/build-deb-cloudberry.yml) workflow.